File: RP3/Desafio5.cpp
#include <iostream>
using namespace std;

int main() {
    string senhaCorreta = "senha123";
    string senha = "";

    while (senha != senhaCorreta) {
        cout << "Digite a senha: ";
        cin >> senha;
    }

    cout << "Bem-vindo! Acesso liberado." << endl;

    return 0;
}

File: RP3/Desafio5.java
import java.util.Scanner;

public class Desafio5 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String senhaCorreta = "senha123";
        String senha = "";

        while (!senha.equals(senhaCorreta)) {
            System.out.print("Digite a senha: ");
            senha = scanner.nextLine();
        }

        System.out.println("Bem-vindo! Acesso liberado.");

        scanner.close();
    }
}